app/config/events.php

```diff
@@ -261,29 +261,28 @@
         'update' => [
             '$description' => 'This event triggers when a message is updated.',
         ],
-        'topics' => [
-            '$model' => Response::MODEL_TOPIC,
+    ],
+    'topics' => [
+        '$model' => Response::MODEL_TOPIC,
+        '$resource' => true,
+        '$description' => 'This event triggers on any topic event.',
+        'create' => [
+            '$description' => 'This event triggers when a provider is created.',
```

Comment:
The descriptions here should mention topic instead of provider

Reply:
done this